pkg/sql/logictest/testdata/logic_test/pg_catalog

```diff
@@ -1373,6 +1373,22 @@ oid    typname  typowner  typlen  typbyval  typtype
 
 user root
 
+query OTOOIBT colnames
+SELECT oid, typname, typnamespace, typowner, typlen, typbyval, typtype
+FROM pg_catalog.pg_type
+WHERE oid = 1000
+----
+oid     typname        typnamespace  typowner  typlen  typbyval  typtype
+1000    _bool          1307062959    NULL      -1      false     b
+
+query OTOOIBT colnames
+SELECT oid, typname, typnamespace, typowner, typlen, typbyval, typtype
+FROM pg_catalog.pg_type
+WHERE oid = 100063
+----
+oid     typname        typnamespace  typowner  typlen  typbyval  typtype
+100063  newtype1       1307062959    NULL      -1      false     e
```

Comment:
Add a test here where we look up an oid that doesnt exist and ensure that we don't error out

Reply:
Done.